shopfloor/services/delivery.py

```diff
@@ -256,21 +293,35 @@ def _lines_from_package_domain(self, package, no_qty_done=True):
             [self._lines_base_domain(no_qty_done), [("package_id", "=", package.id)]]
         )
 
-    def _deliver_product(self, picking, product, product_qty=None):
+    def _deliver_product(self, location, picking, product, product_qty=None):
         if product.tracking in ("lot", "serial"):
```

Comment:
move it to the end to respect other signature and - maybe - make it kw arg (also not to break existing overrides if any)